Tutorial/Window/co_040000.csv

```diff
@@ -0,0 +1,3 @@
+Title#0,"""Dash Panels"""
+text01#0,"""Press the<%Access> to access the dash panels.<br>This will allow you to dash for a limited time."""
```

Comment:
Should this really say "Press the <%Access> to[...]"? I assume that variable finds the keybinding the player has set, so this would result in "Press the E to[...]". So I would think it should say "Press <%Access> to[...]" instead.